conversational-client/src/app/components/mode-selector/mode-selector.component.ts

```diff
@@ -0,0 +1,20 @@
+import {Component} from '@angular/core';
+import {ChatService} from '@services/chat.service';
+
+@Component({
+  selector: 'app-mode-selector',
+  templateUrl: './mode-selector.component.html',
+  styleUrl: './mode-selector.component.css',
+})
+export class ModeSelectorComponent {
+  // The server expects a mode param that matches the lowercase version of one of these string
+  modes = ['Default', 'Fake', 'Gemini', 'History Tutor'];
```

Comment:
nit: should we use a typed string or enum? 
ie. `type PromptMode = 'default' | 'fake' ...`

also, should we define only the ones we support right now?

Reply:
done.